### PR TITLE
[WebRTC] Add descrition and payload APIs

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

- Add GetLocalDescriptino and GetRemoteDescription APIs
- Add PayloadType APIs for RTP payload type

### API Changes ###

 - TCSACR-592

### CODE REVIEW

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs [hunk 1]
```diff
@@ -552,6 +552,58 @@ public int EncoderBitrate
...
+
+            NativeWebRTC.SetPaylodType(WebRtc.Handle, SourceId.Value, codec, payloadType).
+                ThrowIfFailed("Failed to set payload type");
+        }
+
+        /// <summary>
+        /// Gets the RTP payload type of given <paramref name="codec"/>.
+        /// </summary>
+        /// <param name="codec">The transceiver codec.</param>
+        /// <value>The RTP payload type.</value>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:589]: `<value>` is used for properties, the return value of a method is documented with `<returns>`,
same for the other 2 new methods ( Get*Description() )
```suggestion
        /// <returns>The RTP payload type.</returns>
```

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs [hunk 2]
```diff
@@ -318,14 +318,39 @@ public async Task<string> CreateAnswerAsync()
...
             ValidationUtil.ValidateIsNullOrEmpty(description, nameof(description));
 
-            NativeWebRTC.SetLocalDescription(Handle, description).ThrowIfFailed("Failed to set description.");
+            NativeWebRTC.SetLocalDescription(Handle, description).ThrowIfFailed("Failed to set local description.");
+        }
+
+        /// <summary>
+        /// Gets the session description for a local peer.
+        /// </summary>
+        /// <value>The local session descriptino string</value>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:335]: typo, same at 383

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs [hunk 3]
```diff
@@ -318,14 +318,39 @@ public async Task<string> CreateAnswerAsync()
...
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <seealso cref="SetLocalDescription"/>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:337]:
```suggestion
        /// <seealso cref="SetLocalDescription()"/>
```

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs:589]: I'll fix it.

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:335]: I'll fix it.

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs (hunk 3 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:337]: I'll fix it.

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs [hunk 4]
```diff
@@ -318,14 +318,39 @@ public async Task<string> CreateAnswerAsync()
...
             ValidationUtil.ValidateIsNullOrEmpty(description, nameof(description));
 
-            NativeWebRTC.SetLocalDescription(Handle, description).ThrowIfFailed("Failed to set description.");
+            NativeWebRTC.SetLocalDescription(Handle, description).ThrowIfFailed("Failed to set local description.");
+        }
+
+        /// <summary>
+        /// Gets the session description for a local peer.
+        /// </summary>
+        /// <returns>The local session descriptino string</returns>
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:335]: the typo "descriptino" is still here, same in line 383

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs (hunk 4 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs:335]: I'll fix it.

**Pull request merged**